cves/CVE-2011-2729.yml

```diff
@@ -99,7 +99,7 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
+  answer: Automated unit test were not involed in this vulnerability
```

Comment:
How do you know this?

Reply:
There is no evidence of unit tests. There was no test code associated with this check-in.